### PR TITLE
deprecation: IonQ Harmony

### PR DESCRIPTION
**Pull request by speller26**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/advanced_circuits_algorithms/Grover/Grover.ipynb [hunk 1]
```diff
@@ -133,11 +133,6 @@
    ]
```
Comment (virajvchaudhari) [inline examples/advanced_circuits_algorithms/Grover/Grover.ipynb:133]: <p>Harmony appearing here:  <code>Quantum Gates supported by Harmony</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/619/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (virajvchaudhari) [inline examples/advanced_circuits_algorithms/Grover/Grover.ipynb:133]: <p>Same, will need to re-run all cells. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/619/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

File: examples/advanced_circuits_algorithms/Randomness/Randomness_Generation.ipynb [hunk 2]
```diff
@@ -104,10 +104,10 @@
    "outputs": [],
```
Comment (virajvchaudhari) [inline examples/advanced_circuits_algorithms/Randomness/Randomness_Generation.ipynb:104]: <p>Here</p><p> </p>

<pre class="ql-syntax" spellcheck="false">{'arn:aws:braket:us-west-1::device/qpu/rigetti/Aspen-M-3': {'shots': 141, 'tasks': {'CREATED': 1}}, 'arn:aws:braket:us-east-1::device/qpu/ionq/Harmony': {'shots': 141, 'tasks': {'CREATED': 1}}}
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/619/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

File: examples/getting_started/2_Running_quantum_circuits_on_QPU_devices/2_Running_quantum_circuits_on_QPU_devices.ipynb [hunk 3]
```diff
@@ -96,17 +96,17 @@
     "print('\\n') \n",
```
Comment (virajvchaudhari) [inline examples/getting_started/2_Running_quantum_circuits_on_QPU_devices/2_Running_quantum_circuits_on_QPU_devices.ipynb:96]: <p>Should be Ankaa-2 and Harmony. </p><p> </p>

<pre class="ql-syntax" spellcheck="false">Quantum Task Summary
{'arn:aws:braket:us-west-1::device/qpu/rigetti/Aspen-M-3': {'shots': 2000, 'tasks': {'COMPLETED': 2}}, 'arn:aws:braket:us-east-1::device/qpu/ionq/Harmony': {'shots': 1000, 'tasks': {'COMPLETED': 1}}}
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/619/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

File: examples/pennylane/0_Getting_started/0_Getting_started.ipynb [hunk 4]
```diff
@@ -262,7 +262,9 @@
    "outputs": [
```
Comment (virajvchaudhari) [inline examples/pennylane/0_Getting_started/0_Getting_started.ipynb:262]: <pre class="ql-syntax" spellcheck="false">Line #5.    # device_arn = Devices.Rigetti.AspenM3
</pre><p>Should we be removing this as well? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/619/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

File: examples/hybrid_jobs/7_Running_notebooks_as_hybrid_jobs/Running_notebooks_as_hybrid_jobs.ipynb [hunk 5]
```diff
@@ -230,9 +230,9 @@
    "outputs": [
```
Comment (virajvchaudhari) [inline examples/hybrid_jobs/7_Running_notebooks_as_hybrid_jobs/Running_notebooks_as_hybrid_jobs.ipynb:230]: <p>Why are we not replacing AspenM3 with Ankaa here? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/619/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

**Comment by virajvchaudhari** [comment]
Are we also planning to replace Aspen-M3 with Ankaa-2 everywhere?

**Comment by speller26** [comment]
> Are we also planning to replace Aspen-M3 with Ankaa-2 everywhere?

That'll be another PR, as it's more involved; this one only makes changes in examples where Harmony appears.

**Comment by virajvchaudhari** [comment]
> > Are we also planning to replace Aspen-M3 with Ankaa-2 everywhere?
> 
> That'll be another PR, as it's more involved; this one only makes changes in examples where Harmony appears.

Sounds good, just wanted to make sure that we don't miss out [changes to any of our integ_tests](https://github.com/search?q=repo%3Aamazon-braket%2Famazon-braket-examples+Aspen&type=code) as part of the arn changes for AspenM3 in this PR.

**Review by virajvchaudhari — APPROVED** [review]

**Pull request merged**